docker-compose.yaml

```diff
@@ -80,6 +80,16 @@ services:
       - '--settings=${SETTINGS:-/home/kernelci/config/kernelci.conf}'
       - 'run'
 
+  fstests-runner:
+    <<: *base-service
+    container_name: 'kernelci-pipeline-fstests_runner'
+    command:
+      - '/usr/bin/env'
+      - 'python3'
+      - '/home/kernelci/pipeline/fstests_runner.py'
+      - '--settings=${SETTINGS:-/home/kernelci/config/kernelci.conf}'
+      - 'run'
+
```

Comment:
Please drop this, we can't run `kvm-xfstests` inside a container.  The script will be run manually directly on the host instead.

Reply:
updated